xbmc/cores/VideoPlayer/DVDCodecs/Video/AMLCodec.cpp

```diff
@@ -247,12 +242,6 @@ static const int64_t INT64_0 = 0x8000000000000000ULL;
 #ifndef CODEC_TAG_VC_1
 #define CODEC_TAG_VC_1  (0x312D4356)
 #endif
-#ifndef VFORMAT_VP9
-#define VFORMAT_VP9    VFORMAT_UNSUPPORT
-#endif
-#ifndef VIDEO_DEC_FORMAT_VP9
-#define VIDEO_DEC_FORMAT_VP9    VIDEO_DEC_FORMAT_MAX
-#endif
```

Comment:
what is the reason that this definition is moved into header file?
For me it seemed correct that it is defined here because it must not be pulished through the header.

I followed the discussion in LE that there was not found a solution for checking existance for an enum value, bit IMO we should repair the thing right here instead moving to header and choosing not correct values